validphys2/src/validphys/coredata.py

```diff
@@ -284,10 +284,20 @@ def with_cuts(self, cuts):
         new_commondata_table = self.commondata_table.loc[cuts]
         return dataclasses.replace(self, ndata=newndata, commondata_table=new_commondata_table)
 
+    @property
+    def kinematics(self):
+        return self.commondata_table[KINNAMES]
+
+    def get_kintable(self):
+        return self.kinematics.values
```

Comment:
Both this (and the one below) are left as a method because they were used for the old commondata as methods (with those names).

I prefer to leave them like that while the `LegacyCommonData` still exist.